packages/connect/src/data/config.ts

```diff
@@ -36,6 +36,16 @@ export const config = {
             label: 'MetaMask',
             icon: '',
         },
+        {
+            origin: 'bpcdaglidgnlggelgbjfagekoapjmccp',
+            label: 'Rainbow',
+            icon: '',
+        },
```

Comment:
I see the other one is the id for the rainbow extension in chromewebstore, but this one what is for other extension store?

Reply:
Ok, it is used for Chrome store dev build.

Reply:
maybe lets use a different label for dev build, something like "Rainbow DEV"

Reply:
Sounds good, done